### PR TITLE
Network: Update user agents

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: addOns/network/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Minor tweaks in help pages for better rendering.
 - Promoted to Beta status.
 - Maintenance changes.
+- Update user agents
```
Comment (kingthorin) [inline addOns/network/CHANGELOG.md:17]:
```suggestion
- Update user agents.
```

**Review by kingthorin — APPROVED** [review]

File: addOns/network/src/main/resources/org/zaproxy/addon/network/internal/client/common-user-agents.txt [hunk 2]
```diff
@@ -1,68 +1,64 @@
 # Most common user agents c/o https://techblog.willshouse.com/2012/01/03/most-common-user-agents/
-# Last Updated: Fri, 13 May 2022 14:05:15 +0000
+# Last Updated: Last Updated: Mon, 24 Oct 2022 07:05:27 +0000
```
Comment (kingthorin) [inline addOns/network/src/main/resources/org/zaproxy/addon/network/internal/client/common-user-agents.txt:2]:
```suggestion
# Last Updated: Mon, 24 Oct 2022 07:05:27 +0000
```
Sorry I missed this earlier

**Review by kingthorin — DISMISSED** [review]

File: addOns/network/src/main/resources/org/zaproxy/addon/network/internal/client/common-user-agents.txt [hunk 3]
```diff
@@ -1,68 +1,64 @@
 # Most common user agents c/o https://techblog.willshouse.com/2012/01/03/most-common-user-agents/
-# Last Updated: Fri, 13 May 2022 14:05:15 +0000
+Last Updated: Mon, 24 Oct 2022 07:05:27 +0000
```
Comment (kingthorin) [inline addOns/network/src/main/resources/org/zaproxy/addon/network/internal/client/common-user-agents.txt:2]: Does it need the # ?

Comment (psiinon) [reply addOns/network/src/main/resources/org/zaproxy/addon/network/internal/client/common-user-agents.txt]: It does thanks! Fixed.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**